### PR TITLE
Scheduled Events on VM Property

### PR DESCRIPTION
**Pull request by shpa-microsoft**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json [hunk 1]
```diff
@@ -9889,6 +9889,10 @@
           "type": "integer",
           "format": "int32",
           "description": "Specifies the scale set logical fault domain into which the Virtual Machine will be created. By default, the Virtual Machine will by automatically assigned to a fault domain that best maintains balance across available fault domains.<br><li>This is applicable only if the 'virtualMachineScaleSet' property of this Virtual Machine is set.<li>The Virtual Machine Scale Set that is referenced, must have 'platformFaultDomainCount' &gt; 1.<li>This property cannot be updated once the Virtual Machine is created.<li>Fault domain assignment can be viewed in the Virtual Machine Instance View.<br><br>Minimum api‐version: 2020‐12‐01"
+        },
+        "scheduledEventsProfile": {
```
Comment (Sandido) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json:9893]: Update or add a new Example using this new property.

**Review by Sandido — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json (hunk 1 above)
Comment (shpa-microsoft) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json:9893]: Added new example

**Comment by yungezz** [comment]
hi @ArcturusZhang could you pls confirm impact to GO SDK? thanks

**Review by Sandido — APPROVED** [review]
We are not merging these PRs yet, but this is good to go.

**Comment by ArcturusZhang** [comment]
This PR cann impossibly introduce breaking changes to Go SDK. Some of the reported breaking changes are accumulated from previous PRs, but one of the breaking changes deserves noticing: the package 2021-03-01 was removed.
I assume this comes from the base branch - it is not up-to-date with the master. Previously we released compute/2021-03-01 package with cloud services 2021-03-01 and compute 2020-12-01.

Therefore I assume we could ignore this breaking change alarm. But it would be nice if you could update the base branch up-to-date with master branch @shpa-microsoft @Sandido

**Comment by shpa-microsoft** [comment]
@ArcturusZhang merged the latest and now it should not have breaking changes pertaining to base branch. Thanks.

**Comment by shpa-microsoft** [comment]
@Sandido Please re-review. The SDK Failure is in creating nuget package - unrelated to my changes it seems

**Comment by ArcturusZhang** [comment]
Thank you @shpa-microsoft , now the changes are very clear - only a new property is added therefore breaking changes are all accumulated.

**Comment by msyyc** [comment]
The breaking changes for python are accumulated from previous PRs.

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @shpa-microsoft @Sandido let me know when the PR ready to merge.

**Comment by yungezz** [comment]
> hi @shpa-microsoft @Sandido let me know when the PR ready to merge.

ping again @shpa-microsoft @Sandido

**Comment by yungezz** [comment]
> > hi @shpa-microsoft @Sandido let me know when the PR ready to merge.
> 
> ping again @shpa-microsoft @Sandido

ping again...

**Pull request merged**